model/test_results.go

```diff
@@ -41,6 +45,13 @@ const (
 	parquetDateFormat     = "2006-01-02"
 )
 
+// invalidParquetAlertCount is a temporary count to avoid over-logging when we
+// detect invalid Parquet test results.
```

Comment:
What is the parquet alert intended to do? Is it supposed to tell us to fix the mismatched BSON/Parquet test results or is it just to make us aware that it happened at least a few times?

Reply:
Just to make us aware that it is occurring so we can revert, but it is most likely that if it happens to one file its happening to all since they are serialized in the same way. We don't want to bombard splunk with these messages.